tests/DotNetLightning.Core.Tests/TransactionTests.fs

```diff
@@ -149,7 +149,7 @@ let testList = testList "transaction tests" [
         Expect.equal input.PrevOut.Hash (commitmentTx.GetHash()) "wrong prevout hash"
         let expectedAmount =
             let fullAmount = commitmentSpec.ToLocal.ToMoney()
-            let fee = commitmentTx.GetFee [| fundingScriptCoin |]
+            let fee = commitmentTx.GetFee (Array.singleton (fundingScriptCoin :> ICoin))
```

Comment:
I prefer  `[||]` then `Array.singleton` if there is no reason

Reply:
Array.singleton looks more readable to me

Reply:
BDFL is on my side 😤
https://twitter.com/dsymetweets/status/1479404811548254208?s=20&t=ATAtZxNWAo0O_4ckkL9DDA

Reply:
🤣🤣 🤣  alright change it back @aarani...

Reply:
Done